common/table.js

```diff
@@ -194,6 +198,20 @@
                     scope.vm.search = null;
                 };
 
+                scope.addRecord = function() {
+                    // open a new tab, once focus is back to this page, reload
+                    var newRef = scope.vm.reference.contextualize.entryCreate;
+                    var appLink = newRef.appLink;
+
+                    // open url in a new tab
+                    var window = $window.open(appLink, '_blank');
+                    if (window) {
+                        window.focus();
+                    } else {
+                        $window.location.href = appLink;
```

Comment:
What's the purpose of the else case here? Is it so if the page fails to open a new tab, it will redirect the current one?